docs/source/specs/openapi.json

```diff
@@ -1681,6 +1684,11 @@
         "type": "http",
         "description": "The userid/password is needed when accessing this API externally",
         "scheme": "basic"
+      },
+      "identity_auth": {
+        "type": "apiKey",
+        "in": "header",
```

Comment:
Can we add a description here so it's apparent that this is only necessary when accessing the API directly, outside of cloud.redhat.com?

Reply:
Thanks!